src/Nri/Ui/TextArea/V3.elm

```diff
@@ -88,61 +88,38 @@ contentCreation model =
                 [ InputStyles.input theme model.isInError
+                , Css.boxSizing Css.borderBox
+                , case model.height of
+                    AutoResize minimumHeight ->
+                        Css.minHeight (calculateMinHeight theme minimumHeight)
+
+                    Fixed ->
+                        Css.batch []
                 ]
-            , Attributes.autofocus model.autofocus
-            , Attributes.placeholder model.placeholder
-            , Attributes.attribute "data-gramm" "false" -- disables grammarly to prevent https://github.com/NoRedInk/NoRedInk/issues/14859
-            , Attributes.css
-                (minHeight
-                    ++ [ Css.boxSizing Css.borderBox ]
-                )
+                [ Events.onInput model.onInput
+                , Attributes.id (generateId model.label)
+                , Attributes.autofocus model.autofocus
+                , Attributes.placeholder model.placeholder
+                , Attributes.attribute "data-gramm" "false" -- disables grammarly to prevent https://github.com/NoRedInk/NoRedInk/issues/14859
+                ]
+                [ Html.text model.value ]
             ]
-    in
-    Html.div
-        [ Attributes.css [ Css.position Css.relative ]
-        ]
-        [ case model.height of
-            AutoResize _ ->
-                {- NOTES:
-                   The autoresize-textarea element is implemented to pass information applied to itself to an internal
-                   textarea element that it inserts into the DOM automatically. Maintaing this behavior may require some
-                   changes on your part, as listed below.
-
-                   - When adding an Html.Attribute that is a _property_, you must edit Nri/TextArea.js to ensure that a getter and setter
-                   are set up to properly reflect the property to the actual textarea element that autoresize-textarea creates
-                   - When adding a new listener from Html.Events, you must edit Nri/TextArea.js to ensure that a listener is set up on
-                   the textarea that will trigger this event on the autoresize-textarea element itself. See AutoresizeTextArea.prototype._onInput
-                   and AutoresizeTextArea.prototype.connectedCallback for an example pertaining to the `input` event
-                   - When adding a new Html.Attribute that is an _attribute_, you don't have to do anything. All attributes are
-                   automatically reflected onto the textarea element via AutoresizeTextArea.prototype.attributeChangedCallback
-                -}
-                Html.node "autoresize-textarea"
-                    (sharedAttributes
-                        ++ [ -- setting the default value via a text node doesn't play well with the custom element,
-                             -- but we'll be able to switch to the regular value property in 0.19 anyway
-                             Attributes.defaultValue model.value
```

Comment:
> setting the default value via a text node doesn't play well with the custom element, but we'll be able to switch to the regular value property in 0.19 anyway

❓ Oh! Is this no longer the case? (Seems fine!) What fixed it?

Reply:
In the previous setup we weren't inserting a textarea under the `autoresize-textarea` element, so we had to do something different in order to get the default value in the textarea that it eventually created. In the current setup the textarea is always created by the virtual DOM so we can use the text node in both cases.